hapi-fhir-spring-boot/hapi-fhir-spring-boot-autoconfigure/src/main/resources/application.yml

```diff
@@ -0,0 +1,6 @@
+keycloak:
+   auth-server-url: http://localhost:8181/auth/
```

Comment:
I see that two files were edited for this? Is it clear which is required

Reply:
@ekigamba  This properties file is no longer required. Our main `application.yml `in the spring boot app package is enough, I will remove this one.